modules/end-user-guide/pages/selecting-an-ide.adoc

```diff
@@ -5,13 +5,129 @@
 [id="selecting-an-ide_{context}"]
 = Selecting an IDE
 
-You can select your IDE at when starting a workspace: 
+You can select your preferred IDE when starting a workspace: 
```

Comment:
link:https://theia-ide.org/docs/[Eclipse Theia] is the current default editor for {prod-short} but you can select a different IDE when starting a workspace. Following alternatives to Eclipse Theia are available:

* JetBrains IDE such as link:https://www.jetbrains.com/help/idea/getting-started.html[IntelliJ IDEA] or link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[PyCharm]
* Visual Studio Code - link:https://github.com/Microsoft/vscode/[Open Source ("Code - OSS")]

You can either select the IDE by using the URL parameter `che-editor` or using the `.che/che-editor.yaml` file

If you don't select any editor, the latest stable version of Eclipse Theia opens.